### DISCUSSION TITLE
doc: not clear how to test for one number less than another

### DISCUSSION THREAD

**Discussion by shaunc** [stretchr/testify#1659] [Q&A] [answered]
I assume I want to use `Comparison` somehow, but the documentation is quite unclear. What I want is (equivalent to):

```
  assert.LessThan(t, 3, 4). // passes
  assert.LessThan(t, 4, 3). // fails
```

**Comment by ollien** [comment]
For those who find this when googling (like I was), while `master` _does_ include assertions for this, it is not present in the current release (1.3.0). The documentation reflects `master`, rather than the current release.

If you want to do this in the current release, you can do something like this
```go
// Cannot call this more times than we have values for
assert.True(t, func() bool {
	return readCursor < len(pasteContents)
}())
```

**Comment by kandros** [comment, marked as answer]
You can use `assert.Less`

```
	assert.Less(t, 3, 4) // passes
	assert.Less(t, 4, 3) // fails
```

[doc here](https://godoc.org/github.com/stretchr/testify/assert#Less)

**Comment by ollien** [comment]
@kandros This is not present in the current release, only in master.

**Comment by KianYang-Lee** [comment]
Shouldn'nt this be marked as closed? It's already documented [here](https://pkg.go.dev/github.com/stretchr/testify/assert#Less)
